packages/@react-aria/menu/src/useSubmenuTrigger.ts

```diff
@@ -13,16 +13,14 @@
 import {AriaMenuItemProps} from './useMenuItem';
 import {AriaMenuOptions} from './useMenu';
 import type {AriaPopoverProps, OverlayProps} from '@react-aria/overlays';
-import {FocusableElement, FocusStrategy, KeyboardEvent, PressEvent, Node as RSNode} from '@react-types/shared';
+import {FocusableElement, FocusStrategy, KeyboardEvent, PressEvent} from '@react-types/shared';
 import {RefObject, useCallback, useRef} from 'react';
 import type {SubmenuTriggerState} from '@react-stately/menu';
 import {useEffectEvent, useId, useLayoutEffect} from '@react-aria/utils';
 import {useLocale} from '@react-aria/i18n';
 import {useSafelyMouseToSubmenu} from './useSafelyMouseToSubmenu';
 
 export interface AriaSubmenuTriggerProps {
-  /** An object representing the submenu trigger menu item. Contains all the relevant information that makes up the menu item. */
-  node: RSNode<unknown>,
```

Comment:
possible breaking change now that this is released... should we keep it but make it optional?